docs/resources/useful-resources.md

```diff
@@ -2,11 +2,38 @@
 
 🔔 Looking for [accessibility docs?](../accessibility-docs)
 
-## Standards and resources
+One of our goals is to collect information about accessibility and create documentation to empower the Jupyter community to help us make our software accessible. Please use this section to begin to aggregate relevant accessibility third party standards, guides, and documentation.
 
-One of our goals is to collect information about accessibility and create documentation to empower the Jupyter community
-to help us make our software accessible. Please use this section to begin to aggregate relevant third party standards,
-guides, and documentation related to this:
+## Accessibility guidelines
+
+The World Wide Web Consortium (W3C) publishes a number of international recognized technical guidelines, like the Web Accessibility Initiative (WAI) and XHTML standards. These guidelines assure that a given website can be accessed and consumed by as many people as possible.
+
+### Accessible Rich Internet Applications (WAI-ARIA)
+
+[WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/) provides a route to make Web content and applications more accessible, it specifically targets dynamic content and advanced user interface controls that are normally developed in JavaScript, HTML and other related technologies. A website that doesn't implement this guideline will not be accessible to some users with disabilities, specially the ones that rely on a screen reader or cannot use the mouse.
+
+Is important to remember that links, buttons and form elements are the only elements that can receive focus in HTML. Meaning that lists, paragraphs, `div` or `span`, will never get focus from the keyboard making them not accessible to users that are not using a mouse. This is where WAI-ARIA provides Web developers with techniques to enable keyboard users have a structure, making their websites more navigable. This guideline also include modules for Graphics and Digital Publishing.
+
+More info:
+
+- [WAI-ARIA standard](https://www.w3.org/TR/wai-aria/)
+- [ARIA Authoring Practices Guide (APG)](https://www.w3.org/WAI/ARIA/apg/)
+- [WAI-ARIA 1.1 Standard](https://www.w3.org/TR/wai-aria-1.1/)
+
+### Web Content Accessibility Guidelines (WCAG)
+
+The WCAG is a technical standard that takes care of the information that is in the website. This includes natural information, like text, images and sounds, and code or markup that specify structure and presentation.
```

Comment:
> This may be my ignorance, but I don't know what is meant by natural information. Is there something we should link to to specify it?

I think is the term I found in multiple places that basically groups text, images, sounds, code, markup, etc. I feel is basically any type of information that someone can create, but I don't find any resource to link about it. Should I remove it? and just leave it as,

```suggestion
The WCAG is a technical standard that takes care of the information that is in the website. This includes information like text, images and sounds, and code or markup that specify structure and presentation.
```